test/functional/aggregation_tests.js

```diff
@@ -889,6 +889,7 @@ describe('Aggregation', function() {
                       count = count + 1;
                     },
                     function(err) {
+                      if (err) console.dir({ err });
```

Comment:
Should this still be here?

Reply:
nope! removed (I'm just going to remove the rest without commenting more)